docs/releases/upgrading/upgrade-v6.0.md

```diff
@@ -11,3 +11,111 @@ If unsure, run `invenio-cli install` from inside the instance directory before e
 !!! warning "Backup"
 
     Always backup your database and files before you try to perform an upgrade.
+
+## Upgrade steps
+
+### Upgrade your instance dependencies
+
+Edit your `Pipfile` to bump the RDM version:
+
+```bash
+invenio-cli packages update 6.0.0.dev0
+invenio-cli assets build -d
```

Comment:
before this `pip uninstall invenio-communities`

Reply:
Not needed anymore, communities was put back

Reply:
```suggestion
invenio-cli assets build
```

``-d`` is not needed because it's the default.